iroha/src/lib.rs

```diff
@@ -48,6 +50,24 @@ impl Iroha {
     }
 }
 
+/// Identification of an Iroha's entites. Consists of Entity's name and Domain's name.
+///
+/// # Example
+///
+/// ```
+/// use iroha::Id;
+///
+/// let id = Id::new("gold", "mine");
+/// ```
+#[derive(Clone, Debug, PartialEq, PartialOrd, Ord, Eq, std::hash::Hash, Encode, Decode)]
+pub struct Id(pub String, pub String);
```

Comment:
Consider using an ordinary struct with names for members. It's needed to read the docs right now to understand which string stands for what, it can't be infered from types.

Reply:
Good point.